vertx-pojo-mapper-common/src/main/java/de/braintags/vertx/jomnigate/dataaccess/delete/IDelete.java

```diff
@@ -30,6 +32,13 @@
 public interface IDelete<T> extends IDataAccessObject<T> {
 
   /**
+   * Get the items to be handled
+   * 
+   * @return
+   */
+  Iterator<T> getSelection();
```

Comment:
hmmm - an iterator allows traversing through the members, but does not allow the modification of the members, as long as you don't cast them to something different, or? 

But you are right, the getSelection() is now public, where it was protected previously. 
One idea could be to reset getSelection() to be protected again and to place the BeforeDeleteHandler / AfterDeleteHandler into the package parallel to Delete